# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/automake/hotfix-microblaze.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- automake-1.16.5/lib/config.guess.vanilla	2021-10-31 17:32:24.841028851 +0100
+++ automake-1.16.5/lib/config.guess	2021-10-31 17:33:31.480033604 +0100
@@ -1042,6 +1040,9 @@
     m32r*:Linux:*:*)
 	GUESS=$UNAME_MACHINE-unknown-linux-$LIBC
 	;;
+    microblaze:Linux:*:*)
+	echo "$UNAME_MACHINE"-unknown-linux-"$LIBC"
+	;;
     m68*:Linux:*:*)
 	GUESS=$UNAME_MACHINE-unknown-linux-$LIBC
 	;;
